change-beta/@azure-communication-react-c9223c17-ac3b-4223-af7b-ceda8bc8052b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
the change type should be prerelease, since this is only a beta change (including all beta bug fixes and api changes in beta)